src/test/rpc/AccountTx_test.cpp

```diff
@@ -753,6 +756,85 @@ class AccountTx_test : public beast::unit_test::suite
         }
     }
 
+    void
+    testMPT()
+    {
+        testcase("MPT");
+
+        using namespace test::jtx;
+        using namespace std::chrono_literals;
+
+        auto cfg = makeConfig();
+        cfg->FEES.reference_fee = 10;
+        Env env(*this, std::move(cfg));
+
+        Account const alice{"alice"};
+        Account const bob{"bob"};
+        Account const carol{"carol"};
+
+        MPTTester mptAlice(env, alice, {.holders = {bob, carol}});
+
+        // check the latest mpt-related txn is in alice's account history
+        auto const checkAliceAcctTx = [&](size_t size,
+                                          Json::StaticString txType) {
+            Json::Value params;
+            params[jss::account] = alice.human();
+            params[jss::limit] = 100;
+            auto const jv =
+                env.rpc("json", "account_tx", to_string(params))[jss::result];
+
+            BEAST_EXPECT(jv[jss::transactions].size() == size);
+            auto const& tx0(jv[jss::transactions][0u][jss::tx]);
+            BEAST_EXPECT(tx0[jss::TransactionType] == txType);
+
+            std::string const txHash{
+                env.tx()->getJson(JsonOptions::none)[jss::hash].asString()};
+            BEAST_EXPECT(tx0[jss::hash] == txHash);
+        };
+
+        // alice creates issuance
+        mptAlice.create(
+            {.ownerCount = 1,
+             .holderCount = 0,
+             .flags = tfMPTCanClawback | tfMPTRequireAuth | tfMPTCanTransfer});
+
+        checkAliceAcctTx(3, jss::MPTokenIssuanceCreate);
+
+        // bob creates a MPToken;
+        mptAlice.authorize({.account = bob});
+        checkAliceAcctTx(4, jss::MPTokenAuthorize);
+        env.close();
+
+        // TODO: windows pipeline fails validation for the hardcoded ledger hash
+        // due to having different test config, it can be uncommented after
+        // figuring out what happened
+        //
+        // ledger hash should be fixed regardless any change to account history
+        // BEAST_EXPECT(
+        //     to_string(env.closed()->info().hash) ==
+        //     "0BD507BB87D3C0E73B462485E6E381798A8C82FC49BF17FE39C60E08A1AF035D");
```

Comment:
TODO: some config in [windows CI](https://github.com/XRPLF/rippled/actions/runs/16223681070/job/45810314396) is causing this hardcoded ledger hash check to fail. This is being commented out for now until we figure out what has happened and what config needs to be changed